Makefile

```diff
@@ -0,0 +1,65 @@
+ESLINT = node_modules/.bin/eslint
+NODEUNIT = node_modules/.bin/nodeunit
+XYZ = node_modules/.bin/xyz --repo git@github.com:fantasyland/fantasy-land.git --script scripts/prepublish
+
+BEL = $(shell printf '\007')
+ESC = $(shell printf '\033')
+
+
+.PHONY: all
+all: LICENSE index.d.ts index.js
+
+.PHONY: LICENSE
+LICENSE:
+	cp -- '$@' '$@.orig'
+	sed 's/Copyright (c) .* Fantasy Land/Copyright (c) $(shell git log --date=short --pretty=format:%ad | sort -r | head -n 1 | cut -d - -f 1) Fantasy Land/' '$@.orig' >'$@'
+	rm -- '$@.orig'
+
+index.d.ts: names Makefile
+	awk '{ print "export const " $$1 ": string;" }' '$<' >'$@'
+
+index.js: src/index.js names Makefile
+	# Use BEL character as \n to avoid difficulties with including \n in sed
+	# replacement strings. Use ESC character as ' to avoid "'"'"'" silliness.
+	sed "s;/[*] MAPPING [*]/;$(shell awk '{ printf (NR > 1 ? "," : "") "$(BEL)    " $$1 ": $(ESC)fantasy-land/" $$1 "$(ESC)" }' names)$(BEL)  ;" '$<' \
+	| tr $(BEL) '\n' \
+	| tr $(ESC) "'" \
+	>'$@'
+
+
+.PHONY: lint
+lint:
+	$(ESLINT) \
+	  --config node_modules/sanctuary-style/eslint-es3.json \
+	  --env es3 \
+	  --global module \
+	  --global self \
+	  -- index.js
+	$(ESLINT) \
+	  --config node_modules/sanctuary-style/eslint-es6.json \
+	  --env es6 \
+	  --env node \
+	  --rule 'max-len: [off]' \
+	  -- internal/*.js laws/*.js test/*.js
+	make lint-generated-file-index.d.ts
+	make lint-generated-file-index.js
+
+lint-generated-file-%:
+	rm -- '$*'
+	make -- '$*'
+	git diff --exit-code -- '$*'
```

Comment:
This ensures that __names__, __index.js__, and __index.d.ts__ are all in agreement. `make lint` will fail if a change made to one of these files is not reflected in the other two.